tests/message-extensions/README.md

```diff
@@ -0,0 +1,31 @@
+# Agent: Message Extensions
+
+A Teams app that show cases three message extensions features:
+
+- action commands
+- search commands
+- link unfurling
+
+<!--
+Potential features to showcase
+
+General
+- jit install
+
+Action command
+ - where command context is 'composeBox'
+ - action previews
+
+Handle events
+- 'message.ext.anon-query-link';
+- 'message.ext.select-item';
+- 'message.ext.query-settings-url';
+- 'message.ext.setting';
+- 'message.ext.card-button-clicked';
+-->
```

Comment:
why this comment? consider actually improving readme to show how to use this sample?